ci/log_analyze_setup.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash -e
+#
+# preparation for log analyze
+#
+
+# Note, this need to be consistent with docker bind parameter. Might need to come up with a better sync up solution
+LOCAL_CLIENT_LOG_DIR_PATH_DOCKER=/mnt/workspace/jenkins_rt_logs
+LOCAL_CLIENT_LOG_DIR_PATH=$WORKSPACE/jenkins_rt_logs
+echo "[INFO] LOCAL_CLIENT_LOG_DIR_PATH=$LOCAL_CLIENT_LOG_DIR_PATH"
+echo "[INFO] LOCAL_CLIENT_LOG_DIR_PATH_DOCKER=$LOCAL_CLIENT_LOG_DIR_PATH_DOCKER"
+
+# We probably need this, because the regression test job has been split into multiple sub jobs, leading us to more than one log file
```

Comment:
This is all commented out. Is this on purpose? You say in the comments that it probably needs to be done.

Reply:
Sure, should be deleted!